packages/client/src/client/open-feature-client.ts

```diff
@@ -22,6 +21,7 @@ import { OpenFeature } from '../open-feature';
 import { Provider } from '../provider';
 import { InternalEventEmitter } from '../events/internal/internal-event-emitter';
 import { Client } from './client';
+import { Hook } from '../hooks/hook';
```

Comment:
```suggestion
import { Hook } from './hooks';
```

This kind of thing can sometimes lead to import cycles, for prod code it's usually best to import from the barrel file.

I wonder if we can somehow add linting for this :thinking:

Reply:
Ah added the barrel later and missed that. Thanks!
And yes I think that should not be a problem to lint this, maybe in conjunction with import optimizations.